data/releases/5.1.0.md

```diff
@@ -0,0 +1,1038 @@
+   - Restored support for RISC-V and s390x architecture. 
+   - Restored Cygwin port.
+   - Reduced installation size (50% reduction).
+   - Compressed compilation artefacts (.cmi, .cmt, .cmti, .cmo, .cma files).
+   - More than 10 standard library functions made tail-recursive with Tail-Recursion-Modulo-Cons (TRMC).
+   - More examples in the standard library documentation.
+   - More than 20 improved error messages.
+---
+
+## What's new
+
+Some of the highlights in OCaml 5.1.0 are:
+
+   - Many runtime performance regression fixes (dynlinking, weak array, weak hash sets, GC with idle domains, GC prefetching).
+   - Restored support for RISC-V and s390x architectures.
+   - Restored Cygwin port.
+   - Reduced installation size (50% reduction)
+   - Compressed compilation artefacts (.cmi, .cmt, .cmti, .cmo, .cma files)
+   - More than 10 standard library functions made tail-recursive with Tail-Recursion-Modulo-Cons (TRMC)
+   - More examples in the standard library documentation
+   - More than 20 improved error messages
+
+OCaml 5.1.0 is still a relatively experimental release compared to the OCaml
+4.14 branch. In particular
+
+   - The POWER port is being tested in the dev version of the compiler.
+   - The Windows MSVC port is still unavailable.
+   - Ephemeron performances need to be investigated.
+   - GC compaction is a work in progress.
+   - statmemprof is a work in progress.
+   - There are a number of known runtime concurrency bugs (that triggers under
+     rare circumstances)
+
+For a comprehensive list of changes and details on all new features,
+bug fixes, optimizations, etc., please consult the
+[changelog](#Changes).
+
+---
+
+## Installation Instructions
+
+The base compiler can be installed as an opam switch with the following commands:
+```bash
+opam update
+opam switch create 5.1.0
+```
+
+The source code for the release candidate is also directly available on:
+
+* [GitHub](https://github.com/ocaml/ocaml/archive/5.1.0.tar.gz)
+* [OCaml archives at Inria](https://caml.inria.fr/pub/distrib/ocaml-5.1/ocaml-5.1.0.tar.gz)
+
+
+### Configuration Options
+
+The configuration of the installed opam switch can be tuned with the
+following options:
+
+- ocaml-option-afl: set OCaml to be compiled with afl-fuzz instrumentation
+- ocaml-option-bytecode-only: compile OCaml without the native-code compiler
+- ocaml-option-flambda: set OCaml to be compiled with flambda activated
+- ocaml-option-musl: set OCaml to be compiled with musl-gcc
+- ocaml-option-no-flat-float-array: set OCaml to be compiled with --disable-flat-float-array
+- ocaml-option-static :set OCaml to be compiled with musl-gcc -static
+- ocaml-option-address-sanitizer: set OCaml to be compiled with address sanitizer
+- ocaml-option-leak-sanitizer: set OCaml to be compiled with leak sanitizer
+
+
+For instance, one can install a switch with both `flambda` and the naked-pointer checker enabled with
```

Comment:
No nnp checker option in 5.1, so could replace this with one of the other ones (+fp maybe)